ethereum_history_api/oracles/src/script/prepareNoirFixtures.ts

```diff
@@ -52,11 +54,15 @@ for (const chain in FIXTURES) {
         }
       }
 
-      if (transactionHash) {
-        const txIdx = getTxIdx(block, transactionHash);
-        const txProof = await getReceiptProof(client, block.number, txIdx);
-        await writeFile(join(modulePath, 'receipt_proof.nr'), createReceiptProofFixture(txProof));
+      const txIdx = transactionIdx ?? (transactionHash && getTxIdx(block, transactionHash));
```

Comment:
This line is very hard to understand. Let's remove it and stick with hashes and convert them as we did.